lib/inspec/utils/install_context.rb

```diff
@@ -0,0 +1,55 @@
+module Inspec
+
+  # Hueristics to determine how InSpec was installed.
+  module InstallContextHelpers
+
+    def guess_install_context
```

Comment:
For the first instrumentation point, which covers invocation, the return value of this will be one of the data fields. So a row will include fields like "a `check` invocation from a workstation installation using version 4.18.61 on windows2016 ..."